testkit-java/src/main/java/com/akkaserverless/javasdk/testkit/ActionResult.java

```diff
@@ -20,15 +20,15 @@
 import java.util.List;
 import java.util.NoSuchElementException;
 import com.akkaserverless.javasdk.ServiceCall;
-import java.util.concurrent.CompletableFuture;
+import java.util.concurrent.CompletionStage;
 import com.akkaserverless.javasdk.action.Action;
 
 public interface ActionResult<T> {
 
   /** @return true if the call had an effect with a reply, false if not */
   boolean isReply();
 
-  T getReplyMessage();
+  T getReplyMsg();
```

Comment:
I think it should be `getReply()` like in the value entity and es entity `Result` types.

Reply:
ok